packages/chart/src/components/LineChart/index.tsx

```diff
@@ -60,11 +60,25 @@ const LineChart = (props: LineChartProps) => {
           let lineType = config.runtime.series.filter(item => item.dataKey === seriesKey)[0].type
           const seriesData = config.runtime.series.filter(item => item.dataKey === seriesKey)
           const seriesAxis = seriesData[0].axis ? seriesData[0].axis : 'left'
-          let displayArea = legend.behavior === 'highlight' || seriesHighlight.length === 0 || seriesHighlight.indexOf(seriesKey) !== -1
+          let displayArea =
```

Comment:
I'm not seeing any changes in this file, if none can you remove from your commit history?